dev-client/src/context/SoilIdMatchContext.tsx

```diff
@@ -114,6 +115,7 @@ const useLocationCacheFlushing = () => {
   useEffect(() => {
     if (!isOffline && !isFlushed) {
       dispatch(flushLocationCache());
+      dispatch(flushDataCacheErrors());
```

Comment:
Actually, should we be caching matches for errored locations at all? Our messaging to the user is "Try again later" if soil id returns an error -- so wouldn't this only allow them to try again if they happened to go offline and then back online?

Reply:
This executes if the user starts the app in online mode, not just if they go offline -> online. We do need to cache error entries since we don't want to be continuously reloading them.